Tests/DataXScenarios/DataXScenarios/SaveAndDeploy.cs

```diff
@@ -82,28 +100,28 @@ public static StepResult GenerateConfigs(ScenarioContext context)
 
         [Step("restartJob")]
         public static StepResult RestartJob(ScenarioContext context)
-        {            
+        {
             var baseAddress = $"{context[Context.ServiceUrl] as string}/api/DataX.Flow/Flow.ManagementService/flow/restartjobs";
-            string json = JsonConvert.SerializeObject((string)context[Context.FlowName]);            
+            string json = JsonConvert.SerializeObject((string)context[Context.FlowName]);
             string jsonResult = Request.Post(baseAddress,
                     new RequestContent(Encoding.UTF8.GetBytes(json), "application/json"),
                     context[Context.AuthToken] as string);
             dynamic result = JObject.Parse(jsonResult);
-            context[Context.RestartJobsName] = (string)result.result.IsSuccess;            
+            context[Context.RestartJobsName] = (string)result.result.IsSuccess;
             string restartJobsName = context[Context.RestartJobsName] as string;

```

Comment:
same as above. don't need to create restartJobsName

Reply:
restartJobsName is being used in two places. That is why this string is created.